.github/spack/build_spack_package.sh

```diff
@@ -42,7 +48,10 @@ if [ -n "${MATRIX_SPACK_PYTHON}" ]; then
 fi
 
 # the ubuntu CI runner runs on multiple cpu archs; compile for an old one
-ARCH="linux-ubuntu20.04-x86_64"
+ARCH="linux-ubuntu22.04-x86_64_v4"
```

Comment:
Do we want to test multiple runners? This could go into the workflow file…

Reply:
Might make sense but maybe not now. One big new things to test is M1 / Arm support.